proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_hu_HU.properties

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2018 Velocity Contributors
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=Már kapcsolódva vagy ehhez a szerverhez!
+velocity.error.already-connected-proxy=Már kapcsolódva vagy ehhez a proxyhoz!
+velocity.error.already-connecting=Már megpróbálsz kapcsolódni egy szerverhez!
+velocity.error.cant-connect=Nem lehet csatlakozni a(z) {0} szerverhez: {1}
+velocity.error.connecting-server-error=Nem tudunk csatlakoztatni a {0} szerverhez. Kérlek próbáld újra később.
```

Comment:
In all other cases servers are referred to as `a(z) {0} szerver...`. This is an exception, is it intentional?

Reply:
Opps my bad, fixing right now!